float64 Gammam
float64 lamda
float64[3] KD      
float64[3] KDTu
float64[3] GammamTu
float64[3] lamdaTu